src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/entity/EventToRaise.kt

```diff
@@ -14,7 +14,7 @@ import uk.gov.justice.digital.hmpps.manageoffencesapi.enum.EventType
 data class EventToRaise(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  val id: Long = -1,
+  val id: Long = 0,
```

Comment:
after the upgrade was getting hibernate exceptions in the tests relating to -1, had to change to 0